tests/test_oml/test_datasets/test_list_dataest.py

```diff
@@ -0,0 +1,57 @@
+import csv
+from pathlib import Path
+from typing import Iterator, List, Tuple
+
+import pytest
+import torch
+from torch.utils.data import DataLoader
+
+from oml.const import MOCK_DATASET_PATH
+from oml.datasets.list_ import ListDataset
+
+
+@pytest.fixture
+def images() -> Iterator[List[Path]]:
+    yield list((MOCK_DATASET_PATH / "images").iterdir())
+
+
+def get_images_and_boxes() -> List[Tuple[Path, int, int, int, int]]:
```

Comment:
let's use `pd.read_csv` and then call `df['x1']..` instead of unpacking based on positions, which is dangerous and less readable